### ISSUE TITLE
2nd Release - M101.0.4921.0 - AVX2 problem

### ISSUE THREAD

**Issue by paulatreides-web** [Alex313031/Thorium-AVX2#3]
problem with youtube under chromium M101.0.4921.0 -AVX2 (with compatibility mode windows 8)
![image](https://user-images.githubusercontent.com/100779266/156927633-333ec82f-5504-43dd-8f0b-1fe7cca1cc25.png)

**Comment by einsnulleins** [comment]
I might have an issue related to this.

I frequently open bookmark folders with 30 and more entries and this release can't handle it. In 4865 it's fine.

**Comment by Alex313031** [comment]
@paulatreides-web Try windows 7 compat mode.
And @einsnulleins yes I've encountered this too with M101 as a whole. I think its upstream. Consequence of Thorium being based on tip of tree Chromium.

**Comment by gz83** [comment]
> 

When will the upstream fix this problem?

**Comment by gz83** [comment]
> @paulatreides-web Try windows 7 compat mode. And @einsnulleins yes I've encountered this too with M101 as a whole. I think its upstream. Consequence of Thorium being based on tip of tree Chromium.

When will the upstream fix this problem?

**Comment by Lehner82** [comment]
> @paulatreides-web Try windows 7 compat mode.

No difference to W8 for me, 101 is practically unusable.

**Comment by Alex313031** [comment]
@Lehner82 Hmm. Are you using a new or old Chromium profile, and what windows version are you on?

**Comment by Lehner82** [comment]
W10 21H2 with old profile

**Comment by Lehner82** [comment]
I'm gonna try again with a new profile

**Comment by gz83** [comment]
> 



> @Lehner82嗯。您使用的是新的还是旧的 Chromium 配置文件，您使用的是哪个 Windows 版本？

Now the browser cannot be used normally through win7 compatibility mode and win8 compatibility mode, and the browser automatically exits after opening for a while.

In addition, I have also tried changing the configuration file, but the problem is still not solved.

The browser version is the latest 4921.

**Comment by Alex313031** [comment]
@Lehner82 How did it go after a new profile?
@gz83 What do you mean it doesn't work with compat mode?
And what configuration file?

**Comment by gz83** [comment]
> 



> @Lehner82 How did it go after a new profile? @gz83 What do you mean it doesn't work with compat mode? And what configuration file?

"What do you mean it doesn't work with compat mode?"
This problem only appeared in the past few days. I applied the compatibility mode of win7 and win8 respectively, but I couldn't make the browser work normally.

"And what configuration file?"
This refers to user data (User Data).

**Comment by gz83** [comment]
> @gz83 I'm sorry you are having issues. I don't really know what to do from here though, and it works fine for me on win 7 and win 11. Try next release when it comes out and lemme know if it still is an issue. Also about it crashing when opening a lot of tabs from bookmarks (or session buddy, which I found out myself) I have no idea. It has been an issue since M99

I have found something similar to yours before, and I hope the upstream can fix it as soon as possible. Thank you very much for your efforts to bring us a useful browser.

**Comment by Alex313031** [comment]
@gz83 Thanks for saying thanks lol. Also I like your PFP what is it from? Also, I will be building a new windows release tonight and will test in a win7, win10, and win11 VM with both fresh and previously used user profile dirs. If I don't come across any issues, I will publish it, and then you can test and lemme know here how it works for you.

**Comment by Alex313031** [comment]
@gz83 Yeah I mean your avatar. And yeah I'm actually building the AVX version as we speak, and AVX2 right after.

**Comment by gz83** [comment]
> @gz83 Yeah I mean your avatar. And yeah I'm actually building the AVX version as we speak, and AVX2 right after.

My avatar is the mascot of Zhihu, and it is also an arctic fox. I use it as my avatar because I think it looks good. Zhihu is a question-and-answer site, similar to Quora.

**Comment by crocer211** [comment]
@gz83 @Alex313031 compat mode for me seems to disable itself after a certain amount of time for me

**Comment by gz83** [comment]
> @gz83 @Alex313031 compat mode for me seems to disable itself after a certain amount of time for me

Yes, I also found this problem.

**Comment by crocer211** [comment]
@gz83 what i learned to keep it in compat mode is to set it for all users which requires admin rights

**Comment by Alex313031** [comment]
@gz83 @crocer211 @einsnulleins @Lehner82 What you should do is not set the shortcut, but go to the actual executable at C:\Users\$USERNAME\AppData\Local\Chromium\chrome.exe and set compatibility mode there for ALL users.

**Comment by gz83** [comment]
> @gz83 @crocer211 @einsnulleins @Lehner82 What you should do is not set the shortcut, but go to the actual executable at C:\Users$USERNAME\AppData\Local\Chromium\chrome.exe and set compatibility mode there for ALL users.

The settings have been made according to your method. At present, the browser is running normally, and any problems will be reported in time.

**Comment by crocer211** [comment]
> @gz83 @crocer211 @einsnulleins @Lehner82 What you should do is not set the shortcut, but go to the actual executable at C:\Users$USERNAME\AppData\Local\Chromium\chrome.exe and set compatibility mode there for ALL users.

 i had learned that on my own but it will still work if done on the shortcut with these instructions

**Comment by Alex313031** [comment]
@Neucher @einsnulleins @crocer211 @gz83 @paulatreides-web Updating this comment.

Thorium now installs to 
C:\Users$USERNAME\AppData\Local\Thorium\Application\

And the user profile dir is at 
C:\Users$USERNAME\AppData\Local\Thorium\User Data\

So set compatibility mode to Win 7 or 8 for ALL USERS at
C:\Users$USERNAME\AppData\Local\Thorium\Application\chrome.exe

**Comment by gz83** [comment]
> @Neucher @einsnulleins @crocer211 @gz83 @paulatreides-web Updating this comment.
> 
> Thorium now installs to C:\Users$USERNAME\AppData\Local\Thorium\Application\
> 
> And the user profile dir is at C:\Users$USERNAME\AppData\Local\Thorium\User Data\
> 
> So set compatibility mode to Win 7 or 8 for ALL USERS at C:\Users$USERNAME\AppData\Local\Thorium\Application\chrome.exe

OK, got it.
